cypress/support/form/selectFree.js

```diff
@@ -0,0 +1,6 @@
+const selectFree = (name, value) => {
+    cy.get(`[name="${name}"]`).type(value)
+    return cy
+}
+
+Cypress.Commands.add('selectFree', selectFree)
```

Comment:
The other form helpers use the type of input they're chaning in their names. The term `free` does not give any value, I have no idea what it means from reading the code. What do you mean by that?

Reply:
`Free` was supposed to refer to `free text input`, but I now see how that might be confusing. Changed to `selectText`.